plugins/exporter.koplugin/base.lua

```diff
@@ -23,6 +23,7 @@ function BaseExporter:_init()
     self.extension = self.extension or self.name
     self.is_remote = self.is_remote or false
     self.version = self.version or "1.0.0"
+    self.sharable = self.sharable or false
```

Comment:
Remove this one, it is not used.

You call the property `shareable`elsewhere. And the property is not required by any of the BaseExporter methods.

So it is ok if is not defined here.

Reply:
Done.